src/servers/index.files.d.ts

```diff
@@ -0,0 +1,11 @@
+type GraphQLContext = {
+  logger: Logger
+  uid: UserId | null
+  wallet: Wallet | null
+  domainUser: User | null
+  domainAccount: Account | null
+  user: User | null
+  geetest: GeetestType
+  account: Account | null
+  ip: string | undefined
```

Comment:
One nit, we can probably make an `Ip` type for here

Reply:
I think this should be `IpAddress` IIRC